src/main/java/org/eulu/bookshop/controller/BookController.java

```diff
@@ -0,0 +1,35 @@
+package org.eulu.bookshop.controller;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.eulu.bookshop.dto.BookDto;
+import org.eulu.bookshop.dto.CreateBookRequestDto;
+import org.eulu.bookshop.service.BookService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/books")
```

Comment:
add `server.servlet.context-path=/api` to application.properties
 
```suggestion
@RequestMapping("/books")
```